src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -44,121 +58,181 @@ export function formatDate(date) {
  * @param {Object} details optional, properties vary depending on the status type
  * @returns {Contents}
  */
-export function getStatusContents(type, details) {
-  const { nod, awaitingHearingDate, bvaDecision, remand, pendingForm9, onDocket } = STATUS_TYPES;
+export function getStatusContents(statusType, details) {
   const contents = {};
-  if (type === nod) {
-    const office = details.regionalOffice || 'Regional Office';
-    contents.title = `The ${office} is reviewing your appeal`;
-    contents.description = (
-      <p>The {office} received your Notice of Disagreement and is revewing
-      your appeal. This means they review all of the evidence related to your appeal, including
-      any new evidence you submit. They may contact you to request additional evidence or
-      medical examinations, as needed. When they have completed their review, they will
-      determine whether or not they can grant your appeal.</p>
-    );
-  } else if (type === awaitingHearingDate) {
-    const hearingType = details.hearingType || 'hearing';
-    const currenltyHearing = details.currentlyHearing || 'an earlier month';
-    contents.title = 'You are waiting for your hearing date';
-    contents.description = (
-      <p>You have selected to have a {hearingType} in your form 9.
-      Currently the Board is having hearings for appeals of {currenltyHearing}</p>
-    );
-  } else if (type === bvaDecision) {
-    const { decisionIssues } = details;
-    const allowedIssues = decisionIssues
-      .filter((issue) => (issue.disposition === 'allowed'))
-      .map((issue, i) => (<li key={`allowed-${i}`}>{issue.description}</li>));
-    const deniedIssues = decisionIssues
-      .filter((issue) => (issue.disposition === 'denied'))
-      .map((issue, i) => (<li key={`denied-${i}`}>{issue.description}</li>));
-    const businessDays = details.businessDays;
-    contents.title = 'The Board has made a decision on some of your appeals';
-    contents.description = (
-      <div>
-        <div>
-          The Board of Veterans Appeals has made a decision on some of your appeals.
-          You will receive your decision letter in the mail in {businessDays} business days. Here is an overview
-          of the decision:
-        </div>
-        <br/>
-        <div className="decision-items">
-          <h5 className="allowed-items">Allowed</h5>
-          <ul>{allowedIssues}</ul>
-          <h5 className="denied-items">Denied</h5>
-          <ul>{deniedIssues}</ul>
-        </div>
+  switch (statusType) {
```

Comment:
You could make the case that this is more conditions than is appropriate for a `switch` and we should be using something like an object with each type as keys, but I felt it would get kind of messy given that 1. we would need to do some more work to map the constants properly and 2. a `switch` gives us a default / fall-through case for free.